provisioning/plugins/corim-nitro-decoder/README.md

```diff
@@ -0,0 +1,36 @@
+# Endorsement Store Interface
+
+## Reference Value
+
+```json
+{
+  "scheme": "AWS_NITRO",
+  "type": "REFERENCE_VALUE",
+  "attributes": {
+    "psa.hw-model": "RoadRunner",
+    "psa.hw-vendor": "ACME",
+    "psa.impl-id": "IllXTnRaUzFwYlhCc1pXMWxiblJoZEdsdmJpMXBaQzB3TURBd01EQXdNREU9Ig==",
+    "psa.measurement-desc": 1,
+    "psa.measurement-type": "BL",
+    "psa.measurement-value": "h0KPxSKAPTEGXnvOPPA/5HUJZjHl4Hu9eg/eYMTPJcc=",
+    "psa.signer-id": "rLsRx+TaIXIFUjzkzhokWuGiOa48a/2eeHH35di66Gs=",
+    "psa.version": "2.1.0"
+  }
+}
```

Comment:
Definitely a leftover.
I am currently not transporting any expected PCR values. That's not how Veracruz attestation works (they are embedded in the generated certificate and the client authenticates them. I've thought a bit how it might be done for other systems, but yes, we need to figure out how to do that for more general use-cases.

Reply:
> we need to figure out how to do that for more general use-cases.

👍 definitely an important item for discussion.

To support both use cases we need to create a non-ambiguous signal that the verifier plugin can use to distinguish between "don't bother checking PCRs for this workload" and "no matching PCRs for this workload"